validation-test/compiler_crashers_2_fixed/sr10062.swift

```diff
@@ -0,0 +1,6 @@
+// RUN: not %target-swift-frontend -emit-silgen %s
```

Comment:
Please use - `// RUN: %target-swift-frontend -typecheck %s` since the crash was during type-check phase.

Reply:
@xedin Hmm, in the JIRA report, it crashes in SILGen, although with asserts on, it would crash in the type checker.

Reply:
With assertions on it crashes in `ExprRewritter` which is part of the solver.

Reply:
I guess for a fixed test it doesn't matter…

Reply:
Not that much, but it would be great to be consistent.

Reply:
Sorry, I meant "if the test were still failing, it would be better to use `-emit-silgen`, so that it fails consistently, but since it's been fixed, it's okay if regressions are only caught with assertions on".